python/lsst/ts/salobj/topics/read_topic.py

```diff
@@ -242,8 +255,37 @@ def max_history(self):
         while self._callback_tasks:
             task = self._callback_tasks.pop()
             task.cancel()
+        self._reader.close()
         self._data_queue.clear()
 
+    async def aget(self, timeout=None):
+        """Get the current value, if any, else wait for the next new value.
+
+        Parameters
+        ----------
+        timeout : `float` (optional)
+            Time limit, in seconds. If None then no time limit.
+
+        Returns
+        -------
+        data : `DataType`
+            The data.
```

Comment:
From the DDS system (or the DDS system copied to the internal queue, which is not exactly the same thing). I hope it is clear based on which method is being called.

Reply:
Please expand this docstring with a paraphrase of your above comment. I think that's useful information.

Reply:
I'm really getting lost here. A ReadTopic's whole purpose in life is to read data from DDS. I don't see why I'd have to explain that in docs for methods!

I do try to be clear whether data comes from a queue, and the queue I mean is always the internal queue (which is not the DDS queue). The class doc string has a section describing the two queues. But I consider *that* an internal detail that would be a really bad idea to try to explain in a doc string for a method. I'm not entirely comfortable even putting it in the class doc string, but I figured it might help folks debug problems if a reader starts falling behind or losing messages.

Reply:
At this point the doc string reads:
```
        """Get the current value, if any, else wait for the next value.

        Parameters
        ----------
        timeout : `float` (optional)
            Time limit, in seconds. If None then no time limit.

        Returns
        -------
        data : `DataType`
            The current or next value.
...
```

Reply:
That is better, thank you.

I asked these questions because I'm not at all familiar with these things (I don't even know what a DDS queue is at this point). So "new user getting up to speed" is the audience here.